src/plugins/scene3d/Scene3D.hh

```diff
@@ -72,6 +75,13 @@ namespace plugins
     public: virtual void LoadConfig(const tinyxml2::XMLElement *_pluginElem)
         override;
 
+    /// \brief Callback for saving a screenshot (from the user camera) request
+    /// \param[in] _msg Request message of the saved file path
+    /// \param[in] _res Response data
+    /// \return True if the request is received
+    private: bool OnScreenshot(const msgs::StringMsg &_msg,
+        msgs::Boolean &_res);
```

Comment:
See suggestions on #171 on how to keep all the logic within the Screenshot plugin.